src/application/node/application.ts

```diff
@@ -24,7 +25,8 @@ export class BackendApplication {
 
     private app: express.Application;
 
-    constructor( @inject(ContributionProvider) @named(BackendApplicationContribution) private contributionsProvider: ContributionProvider<BackendApplicationContribution>) {
+    constructor( @inject(ContributionProvider) @named(BackendApplicationContribution) private contributionsProvider: ContributionProvider<BackendApplicationContribution>,
+        @inject(ILogger) private logger: ILogger) {
```

Comment:
please use protected where it is possible, I am not a big fun or `private`, subclasses should be able to access the state of superclasses